alpha-cli-app/build.gradle.kts

```diff
@@ -0,0 +1,64 @@
+plugins {
+    id("alpha.java-application-conventions")
+}
+
+dependencies {
+    implementation(project(":alpha-solver"))
+    implementation(project(":alpha-commons"))
+
+	implementation("commons-cli:commons-cli:1.3.1")
+
+	val poiVersion = "4.1.1"
+	implementation("org.apache.poi:poi:${poiVersion}")
+	implementation("org.apache.poi:poi-ooxml:${poiVersion}")
+}
+
+val main = "at.ac.tuwien.kr.alpha.Main"
+
+application {
+    mainClass.set(main)
+}
+
+tasks.create<Jar>("bundledJar") {
```

Comment:
When running the `bundledJar` task using `gradlew` it fails
```
./gradlew bundledJar
> Task :alpha-cli-app:bundledJar FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':alpha-cli-app:bundledJar'.
> Entry at/ac/tuwien/kr/alpha/api/impl/AlphaImpl$1.class is a duplicate but no duplicate handling strategy has been set. Please refer to https://docs.gradle.org/7.3/dsl/org.gradle.api.tasks.Copy.html#org.gradle.api.tasks.Copy:duplicatesStrategy for details.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 8s
20 actionable tasks: 2 executed, 18 up-to-date
```
It seems that the duplicate originates in some copy operation of the `bundledJar` task.
Is it possible that the `compileClasspath` contains files that are also in the `runtimeClasspath` and hence duplicated?

Reply:
This should be fixed in b07a088b. The mistake was duplicating `with(tasks["jar"] as CopySpec)` which would lead to all files being copied twice.

Reply:
Thanks, now I can build the bundledJar.

However, a follow-up issue: when I run the bundled jar, it shows me some SLF4J warnings as follows:
```
java -jar alpha-cli-app-bundled.jar -i ASPCOMP2013-instances/b27/instances/0001-graph_colouring-125-0.asp 
SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
Answer set 1:
{ ...
```
@lorenzleutgeb any idea what happens here?

Reply:
Yes, we do not have any implementation of a logger in the runtime classpath of `alpha-cli-app`. I'll add one.

Reply:
Fixed in 073da170.